### ISSUE TITLE
On Demand Episode Page is accessible

### ISSUE THREAD

**Issue by pharingee** [bbc/simorgh#5985]
**Describe the solution you'd like**
A11y Acceptance Critria:
Screenreader UX: https://zpl.io/2jR6Axr

Full Page acceptance criteria is documented here for reference: https://docs.google.com/document/d/10rzotmSurDgpsZJgbdxvZ_HOzypJa-qHo5tCuoSr_t0/edit#heading=h.6zanlwthqyxw

**h1**
- Given I am a screenreader user 
And I navigate to an ondemand radio page
When I navigate to the H1 
Then “[Brand Title], [Presentation title]” is announced 
 
**Content order**
- Given I am using a screen reader
When I navigate through the main container
Then the h1 should be announced first
And that should be followed by the paragraph text
And then the audio player  

**Player**
- Given I use a screen reader
And I am on an ondemand radio page on 
When I navigate to the outer iframe of the SMP on the radio page
Then “Audio Player” is announced
(in the language of that service)

- Given I use a screen reader
And I am on an ondemand radio page on 
When I navigate to the inner iframe of the SMP on the radio page
Then the brand and presentation title will be announced

- Given I use a screen reader
And I am on an ondemand radio page on 
When I navigate to the play button of the iframe
then “Play audio, Brand title, Presentation title, Duration 1,00” is announced
(in the language of that service)

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by greenc05** [comment]
That's not all the accessibility acceptance criteria for the page, that's just for the player, please could you add the rest @aeroplanejane

**Comment by aeroplanejane** [comment]
Have added @greenc05, thank you for spotting. 
Acceptance criteria was documented and covered in our amigos session when we reviewed: WSMEDIA-18
As a general rule I do not write github issues, developers do, and we review the tickets in amigos / refinement. 

@rebeccarfmcginnbbc @pharingee in future can we  ensure a11y requirements are covered in the github issues after we discuss them in amigos? I think the best way to do this is to not treat a11y as a separate story but as part of the acceptance criteria for a story if we can, thanks cc @graemelindsayBBC

**Comment by karinathomasbbc** [comment]
In case I miss the amigos on this, can we check on a couple of things:
- Does it make sense to start building the Hero component as part of this piece of work? 
- OR, are we going to ensure that the page & player passes a11y acceptable criteria for now, until we come to the Hero milestone? (https://github.com/orgs/bbc/projects/53#card-33341438 or on Story MAP - https://miro.com/app/board/o9J_kvQbdPo=/?moveToWidget=3074457347267997800&cot=13)

**Comment by pharingee** [comment]
We need to have a 3 amigos on this to see if this depends on the Hero work

**Comment by aeroplanejane** [comment]
It might make more sense to do this alongside the hero image work as we don't want to undo and redo the work - we decided we are definitely including the image in the first MVP, right @gavinspence?

**Comment by gavinspence** [comment]
Yep - we will be including the images on this page as part of the first MVP.

**Comment by karinathomasbbc** [comment]
Done!
